### PR TITLE
Misc fixups for transfers

### PR DESCRIPTION
**Pull request by GiudGiud**
Hoping to get to the big ticket items soon, but small PR for now

### CODE REVIEW

**Comment by mangerij** [comment]
Ferret fail seems _related_ but unsure why since the tests passed.
Note that the only test that uses the multiapp transfer is disabled. I am not sure how turn it on, given that the ./configure step will turn on the SCALFMM (additional module) compile if and only if it sees the headers. Can anyone help me with this logic? thanks

**Comment by GiudGiud** [comment]
It looks like the testing harness crashes at cleanup time. 
It seems to be happening intermittently, since the EM module got merged
https://civet.inl.gov/branch/6695/

**Comment by GiudGiud** [comment]
I can have a look at the SCALFMM testing tomorrow

**Comment by GiudGiud** [comment]
@mangerij the test failure is unrelated to this PR. I think there's an issue with longest jobs printing for very low numbers of tests. I ll make a PR to MOOSE.

**Comment by GiudGiud** [comment]
@fdkong for review pls

File: test/tests/transfers/multiapp_copy_transfer/aux_to_primary/sub.i [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  solve = false
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+[]
+
+[Variables/sub]
+  initial_condition = 1980
```
Comment (fdkong) [inline test/tests/transfers/multiapp_copy_transfer/aux_to_primary/sub.i:12]: Were you born in 1980?

**Review by fdkong — APPROVED** [review]
Looks good to me

File: test/tests/transfers/multiapp_copy_transfer/aux_to_primary/sub.i (hunk 1 above)
Comment (GiudGiud) [reply test/tests/transfers/multiapp_copy_transfer/aux_to_primary/sub.i:12]: nope

**Pull request merged**